hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestNSSummaryEndpointWithLegacy.java

```diff
@@ -389,15 +399,47 @@ public void testUtility() {
 
   @Test
   public void testGetBasicInfoRoot() throws Exception {
+    String username = "myuser";
+    OmPrefixInfo omPrefixInfo = getOmPrefixInfoForTest(ROOT_PATH,
+        IAccessAuthorizer.ACLIdentityType.USER,
+        username,
+        IAccessAuthorizer.ACLType.WRITE,
+        ACCESS);
+    omPrefixInfo.getMetadata().put("key", "value");
+    reconOMMetadataManager.getPrefixTable()
+        .put(OzoneConsts.OM_KEY_PREFIX, omPrefixInfo);
     // Test root basics
     Response rootResponse = nsSummaryEndpoint.getBasicInfo(ROOT_PATH);
     NamespaceSummaryResponse rootResponseObj =
         (NamespaceSummaryResponse) rootResponse.getEntity();
     Assert.assertEquals(EntityType.ROOT, rootResponseObj.getEntityType());
-    Assert.assertEquals(2, rootResponseObj.getNumVolume());
-    Assert.assertEquals(4, rootResponseObj.getNumBucket());
-    Assert.assertEquals(5, rootResponseObj.getNumTotalDir());
-    Assert.assertEquals(10, rootResponseObj.getNumTotalKey());
+    Assert.assertEquals(2, rootResponseObj.getCountStats().getNumVolume());
+    Assert.assertEquals(4, rootResponseObj.getCountStats().getNumBucket());
+    Assert.assertEquals(5, rootResponseObj.getCountStats().getNumTotalDir());
+    Assert.assertEquals(10, rootResponseObj.getCountStats().getNumTotalKey());
```

Comment:
This is a repetitive code. I think moving this to a function would be nicer.

Reply:
Thanks @hemantk-12 for review. Changes done. Pls re-review.

Reply:
My bad, it has been fixed.